spec/factories/user.rb

```diff
@@ -0,0 +1,15 @@
+FactoryGirl.define do
+  factory :user do
+    name 'Test user'
+    role 'user'
+    email 'example@mail.com'
+    password 'passw_1234'
+
+    factory :user_admin do
```

Comment:
you already added `.byebug_history` to `.gitignore` but don't forget to delete this file